src/components/services/UptimeKuma.vue

```diff
@@ -112,13 +112,15 @@ export default {
     },
   },
   created() {
-    /* eslint-disable */
-    this.item.url = `${this.item.url}/status/${this.dashboard}`;
+    // eslint-disable-next-line vue/no-mutating-props
+    this.item.url = this.item.url.endsWith(this.dashboard)
+      ? this.item.url
+      : `${this.item.url}/status/${this.dashboard}`;
```

Comment:
To avoid mutating props, I would suggest to remove all this, remove the slug property (need [documentation](https://github.com/bastienwirtz/homer/blob/main/docs/customservices.md#uptime-kuma) update too), to use the common endpoint property. If you remove those line and configure the service this way:

```yml
- name: "Uptime Kuma"
  url: "http://192.168.0.151:3001"
  endpoint: "http://192.168.0.151:3001/status/default" # Replace default by your dashboard name if any
  type: "UptimeKuma"
``` 

instead of 
```yml
- name: "Uptime Kuma"
  url: "http://192.168.0.151:3001"
  slug: "myCustomDashboard" # Defaults to "default" if not provided.
  type: "UptimeKuma"
```

It should works fine.

Reply:
I agree mutating props isn't great, however your suggestion doesn't seem feasible, as `endpoint` is required to be a base path for the status queries to work https://github.com/bastienwirtz/homer/blob/b168430fbf3f3a85bfcd6aa2d84aab455d4e464b/src/components/services/UptimeKuma.vue#L120-L131